src/main/java/mate/academy/util/ConnectionUtil.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/myDatabase?useSSL=false&serverTimezone=UTC";
```

Comment:
Don't use schema's name in your JDBC URL, as you are configuring the schema while establishing a connection with the database. Make sure the URL is configured correctly without specifying the schema directly in it.